### ISSUE TITLE
Available mem is caculated incorrectly in Linux 2.6.32

### ISSUE THREAD

**Issue by lanixzcj** [shirou/gopsutil#804]
**Describe the bug**
[A clear and concise description of what the bug is.]
Hi，

We simulate  some common fault scenarios.
One of the fault scenarios is injecting specify percent mem load by mounting a tmpfs  and write files in it by `dd`。

we use the powerful `gopsutil` to get total mem and available mem. But the  available mem seems to be calculated incorrectly.

The log for `mem.VirtualMemory()` before we inject mem load fault .
![incorrect avaliable mem1](https://user-images.githubusercontent.com/10002197/71249523-492cfa00-2358-11ea-8341-ab3183018e4c.jpg)
The log for `mem.VirtualMemory()` after we inject mem load fault.
![incorrect available mem2](https://user-images.githubusercontent.com/10002197/71249533-534ef880-2358-11ea-837b-44fc13c8c97a.jpg)

log text:
```
available is 3943956480. total is 4153847808 mem is {total:4153847808,available:3943956480,used:101163008,usedPercent:2.4354047783158452,free:3680378880,active:200273920,inactive:190668800,wired:0,laundry:0,buffers:108728320,cached:263577600,writeback:0,dirty:69632,writebacktmp:0,shared:352256,slab:36982784,sreclaimable:15720448,sunreclaim:21262336,pagetables:2318336,swapcached:0,commitlimit:2076921856,committedas:551084032,hightotal:0,highfree:0,lowtotal:0,lowfree:0,swaptotal:0,swapfree:0,mapped:12095488,vmalloctotal:35184372087808,vmallocused:19210240,vmallocchunk:35184348254208,hugepagestotal:0,hugepagesfree:0,hugepagesize:2097152}
available is 3943522304. total is 4153847808 mem is {total:4153847808,available:3943522304,used:101580800,usedPercent:2.4454627298661014,free:2225438720,active:200318976,inactive:1641897984,wired:0,laundry:0,buffers:108744704,cached:1718083584,writeback:0,dirty:73728,writebacktmp:0,shared:1451581440,slab:40271872,sreclaimable:19005440,sunreclaim:21266432,pagetables:2318336,swapcached:0,commitlimit:2076921856,committedas:2002845696,hightotal:0,highfree:0,lowtotal:0,lowfree:0,swaptotal:0,swapfree:0,mapped:12451840,vmalloctotal:35184372087808,vmallocused:19210240,vmallocchunk:35184348254208,hugepagestotal:0,hugepagesfree:0,hugepagesize:2097152}
```

**To Reproduce**
Inject mem load fault. 
``` shell
`mount -t tmpfs tmpfs filepath -o size=100%`
`dd if=/dev/zero of=filepath ba=1M count=1024`
```
Avaliablde mem does not changed.
``` go
virtualMemory, err := mem.VirtualMemory()
availableMem := virtualMemory.Available
```

**Expected behavior**
[A clear and concise description of what you expected to happen.]
Available mem is caculated correct.
We can inject specify percent mem load fault. 

**Environment (please complete the following information):**
 - [ ] Windows: [paste the result of `ver`]
 - [x] Linux: [paste contents of `/etc/os-release` and the result of `uname -a`]
 - [ ] Mac OS: [paste the result of `sw_vers` and `uname -a`
 - [ ] FreeBSD: [paste the result of `freebsd-version -k -r -u` and  `uname -a`]
 - [ ] OpenBSD: [paste the result of `uname -a`]

`/etc/os-release`: file does not exist
`uname -a`: Linux iZm5efq6ofuv8dswh3qyn8Z 2.6.32-696.16.1.el6.x86_64 #1 SMP Wed Nov 15 16:51:15 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux

**Additional context**
[Cross-compiling? Paste the command you are using to cross-compile and the result of the corresponding `go env`]

**Comment by lanixzcj** [comment]
I found maybe the key `Inactive(file)` in `meminfo` is wrongly written `Inactive(file)` case the bug.

mem_linux.go
line 70
``` go
case "Active":
	ret.Active = t * 1024
case "Inactive":
	ret.Inactive = t * 1024
case "Active(file)":
	activeFile = true
	retEx.ActiveFile = t * 1024
case "In**A**ctive(file)":
	inactiveFile = true
	retEx.InactiveFile = t * 1024
``` 

I found `meminfo` in my system does have `InActive(file)` key but has `Inactive(file)`.

I search the kernel source code, It seems to should be `Inactive(file)`

Linux kernel `fs/proc/meminfo.c`:
``` c
show_val_kb(m, "MemTotal:       ", i.totalram);
show_val_kb(m, "MemFree:        ", i.freeram);
show_val_kb(m, "MemAvailable:   ", available);
show_val_kb(m, "Buffers:        ", i.bufferram);
show_val_kb(m, "Cached:         ", cached);
show_val_kb(m, "SwapCached:     ", total_swapcache_pages());
show_val_kb(m, "Active:         ", pages[LRU_ACTIVE_ANON] +
pages[LRU_ACTIVE_FILE]);
show_val_kb(m, "Inactive:       ", pages[LRU_INACTIVE_ANON] +
pages[LRU_INACTIVE_FILE]);
show_val_kb(m, "Active(anon):   ", pages[LRU_ACTIVE_ANON]);
show_val_kb(m, "Inactive(anon): ", pages[LRU_INACTIVE_ANON]);
show_val_kb(m, "Active(file):   ", pages[LRU_ACTIVE_FILE]);
show_val_kb(m, "Inactive(file): ", pages[LRU_INACTIVE_FILE]);
show_val_kb(m, "Unevictable:    ", pages[LRU_UNEVICTABLE]);
show_val_kb(m, "Mlocked:        ", global_zone_page_state(NR_MLOCK));
```

I create a PR for this iuuse #803 .
Please review.
Thank you~

**Comment by lanixzcj** [comment]
I rename InActive(file) to Inactive(file) in mem_linux.go and test.

Avaliablde mem can be caculated correctly.

Before we inject mem load fault.
![correct avaliablde mem1](https://user-images.githubusercontent.com/10002197/71223512-2e826300-230f-11ea-8a5a-139c49751b85.png)

After we inject mem load fault.
![correct avaliablde mem2](https://user-images.githubusercontent.com/10002197/71223522-36420780-230f-11ea-8493-7c825d322c32.png)

**Comment by shirou** [comment]
closed by #803. Thank you for your contribution!
